templates/pages/overview.twig

```diff
@@ -10,7 +10,7 @@
   <ul>
     {% macro showNamespace(namespace) %}
       <li>
-        <a href="{{ namespace.name|namespace_url }}">{{ namespace.name }}</a>
+        <a href="{{ namespace.qualifiedName|namespace_url }}">{{ namespace.name }}</a>
```

Comment:
Had to change that property just like it is used in the `namespace.twig` so the link gets generated correctly